app/views/layouts/_nav.html.erb

```diff
@@ -0,0 +1,6 @@
+<nav class="splash w-full flex gap-20 text-white p-6 text-xl">
+  <p><%= link_to '<i class="fa-solid fa-bars"></i>'.html_safe, root_path %></p>
```

Comment:
- [ ] Great job working on this capstone. Your app is working well, but we have few things to fix as per project requirements.
- The icon you have used for the back button is not quite representive and does not give a good UI for users. The icon you are using for Back is mostly used for menu. Kindly change this icon.
![Screen Shot 2023-02-21 at 6 17 18 AM](https://user-images.githubusercontent.com/62849872/220370611-2928bbd2-6574-4f23-8c4a-f4985c553b02.png)